x-pack/legacy/plugins/ml/public/jobs/jobs_list/components/jobs_list/jobs_list.js

```diff
@@ -209,7 +230,32 @@ class JobsListUI extends Component {
         }),
         sortable: true,
         truncateText: false,
+        width: '8%'
       }, {
+        name: intl.formatMessage({
```

Comment:
As above, is it possible to hide this column if ML is not enabled in the space?

Reply:
Added as a TODO in a follow-up 👍